ctrlf.el

```diff
@@ -387,7 +399,9 @@ fails, return nil, but still move point."
   ;; Next search should go forward.
   (setq ctrlf--backward-p nil)
   ;; Force recalculation of search.
-  (setq ctrlf--last-input nil))
+  (setq ctrlf--last-input nil)
+  (when ctrlf-auto-recenter
+    (ctrlf-recenter)))
```

Comment:
To avoid duplication, can we do this in the `ctrlf--minibuffer-post-command-hook` instead?

Reply:
Done.